### ISSUE TITLE
about atari 

### ISSUE THREAD

**Issue by xiaolonghao** [werner-duvaud/muzero-general#127]
After training according to the configuration file of Breakout, the effect cannot reach 800+ as stated in the paper. Can anyone give us a result about Atari game, detailed configuration file, thank you.

**Comment by EngrStudent** [comment]
I'm running it at default, and the total reward jumps but before 5k iterations it crashes.  

For games like this it may be that the learning parameters are imperfect, they don't reproduce the results well on different hardware, different operating systems.  If there is a term that reduces the learning rate as a function of iteration count, that might be a decent culprit.

**Comment by xiaolonghao** [comment]
I have adjusted the parameters for several times, and the total reward is all below 10. I feel that the influence of the super parameters is quite great, so I want to find a configuration file that can get normal reward. Do you have any recommendations

**Comment by qianfangjj** [comment]
@xiaolonghao Have you improved the performance of Breakout?

**Comment by ZhengWenZhang** [comment]
@qianfangjj @werner-duvaud @xiaolonghao It seems that the hyper parameter `stacked_observations` in `breakout.py` was set to 0 which doesn't fit the feature of breakout.(btw, it seems that of `atari.py` was set correctly to 32)

**Comment by xiaolonghao** [comment]
> @xiaolonghao您是否提高了 Breakout 的性能？

no.
